charts/kargo/templates/controller/deployment.yaml

```diff
@@ -44,18 +44,32 @@ spec:
         envFrom:
         - configMapRef:
             name: kargo-controller
-        {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd }}
+        {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd .Values.controller.committer.signingKeySecret.name }}
         volumeMounts:
+        {{- if .Values.controller.committer.signingKeySecret.name }}
+        - mountPath: /etc/kargo/committer
+          name: committer
+          readOnly: true
+        {{- end }}
```

Comment:
Nit: I'd prefer this were the last volumeMount so that the order of the volumeMounts is the same as what's on line 47. I know it's such a little thing, but it templates are hard to read, so every little bit helps.

Reply:
Good call, updated:

[refactor(charts): order volumes and mounts more logically](https://github.com/akuity/kargo/pull/1843/commits/5e41a69a00a3ac71e2f51afe7c201dc49119b7c8)